pages/benchmarks.tsx

```diff
@@ -46,7 +46,27 @@ const Benchmarks = () => {
     <>
       <Head>
         <title>Benchmarks - Deno</title>
+
+        <meta name="title" content="Benchmarks - Deno" />
         <meta name="description" content="Continuous benchmarks for Deno." />
+
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta name="twitter:url" content="https://deno.land" />
```

Comment:
This isn't the URL of the benchmarks page?

Reply:
Yes, I'll change that URL to `https://deno.land/benchmarks`